src/wp-includes/script-loader.php

```diff
@@ -121,7 +123,7 @@ function wp_default_packages_vendor( $scripts ) {
 		$scripts->add( $handle, $path, $dependencies, $version, 1 );
 	}
 
-	$scripts->add( 'wp-polyfill', null, array( 'wp-polyfill' ) );
+	$scripts->add( 'wp-polyfill', null, array( 'wp-polyfill-core-js', 'wp-polyfill-regenerator' ) );
```

Comment:
This looks good. We need to be careful to not change the script handle. I don't think the other changes above will have any affect on that. But we should just double check.